muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/lazyframe.js

```diff
@@ -5,45 +5,52 @@
     
     _create : function() {
       $(this.element).removeClass('lazyFrame');
-      $(this.element).waypoint($.proxy(function(direction) {
-        if (direction == 'down') {
-          var url = $(this.element).attr('data-url');
-          if (url) {
-            $(this.element).removeAttr('data-url');
-            $(this.element).attr('src', url);
-            if (window.OPEN_FRAMES === undefined) {
-              window.OPEN_FRAMES = [];
-            }
-            window.OPEN_FRAMES.push(this.element);
-            while (window.OPEN_FRAMES.length > 4) {
-              var frame = window.OPEN_FRAMES.shift();
-              url = $(frame).attr('src');
-              $(frame).attr('data-url', url);
-              $(frame).removeAttr('src');
-            }        
-          }
-        } 
-      }, this), {
-        offset: '100%'
+      
+      $(this.element)      
+      .waypoint({
```

Comment:
Might as well keep this on the same line as $(this.element) so that indentation usually follows {